src/TodoList.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import TodoItem from './TodoItem';
+
+const TodoList = props => (
+  <dl>
+    {props.todos.map(todo => (
```

Comment:
```suggestion
    {todos.map(todo => (
```
Если деструктуризировать в пятой строке из props массив todos, не надо будет перед todos писать props.